eo-parser/src/main/java/org/eolang/parser/XeListener.java

```diff
@@ -158,7 +160,15 @@ public void enterMetas(final ProgramParser.MetasContext ctx) {
 
     @Override
     public void exitMetas(final ProgramParser.MetasContext ctx) {
-        // This method is created by ANTLR and can't be removed
+        if (new MissingEmptyLineAfterMetas().test(ctx.getText())) {
```

Comment:
@mximp looks like a hack to me. Can't we do the same just by using `Program.g4` grammar rules?

Reply:
@yegor256 I didn't mange to invent such grammar change which would keep the grammar unambiguous..